charts/backstage/Chart.yaml

```diff
@@ -38,4 +38,4 @@ sources:
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 2.3.1
```

Comment:
Can we bump a minor version here? I guess you're introducing a new feature here, it's not a bugfix.

Reply:
Yep, have bumped, shall dismiss the review so we can merge if thats ok